samples/Azure.AI.DocumentTranslation/Generated/DocumentTranslationClient.cs

```diff
@@ -65,43 +65,35 @@ public DocumentTranslationClient(string endpoint, AzureKeyCredential credential,
         /// <param name="documentId"> Format - uuid.  The document id. </param>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
         /// <remarks>
+        /// Below is the JSON schema for the request and response payloads.
+        /// Additional information can be found in the service REST API documentation:
+        /// https://docs.microsoft.com/rest/api/AzureAIDocumentTranslation/DocumentTranslation/GetDocumentStatus
         /// Schema for <c>Response Body</c>:
         /// <code>{
-        ///   path: string,
-        ///   sourcePath: string,
-        ///   createdDateTimeUtc: string (ISO 8601 Format),
-        ///   lastActionDateTimeUtc: string (ISO 8601 Format),
-        ///   status: &quot;NotStarted&quot; | &quot;Running&quot; | &quot;Succeeded&quot; | &quot;Failed&quot; | &quot;Cancelled&quot; | &quot;Cancelling&quot; | &quot;ValidationFailed&quot;,
-        ///   to: string,
+        ///   path: string, # Optional. <Description>Location of the document or folder</Description>
```

Comment:
I think `<Description>` tag doesn't work in the docs. See `Scan id` in Remarks section [here](https://review.docs.microsoft.com/en-us/dotnet/api/azure.analytics.purview.scanning.purviewscanclient.createorupdate?view=azure-dotnet-preview&branch=ShivangiTestBranchWithoutFiles#remarks).

Reply:
Hello @ShivangiReja What's the problem? Do you mean that the `<Discription>` tag is not xml tag, and we should not use the tag? and maybe we should use `<p>` instead of `<Discription>`.

Reply:
I meant `<Discription>` tag doesn't work in msdocs as you can see in my generated docs, so we shouldn't use it.

Reply:
Hi @ShivangiReja so shall we just remove `<Discription>` tag or we use some xml valid tag such as `<p>` instead?

Reply:
Yes, we should remove `<Discription>` tag. I think by default, the browser applies a line break before and after the `<p>` tag to separate the paragraphs. You can see it by generating the docs.

Can we add something like below without any tags?
```xml
   ///   path: string, # Optional. Location of the document or folder.
```

Reply:
See here - https://www.w3docs.com/learn-html/html-p-tag.html

>The `<p>` tag defines a paragraph of text. It is a block-level element and always starts on a new line. Before and after each paragraph, browsers add margin automatically. 

We want to add the description on the same line as the property and it's type.

Reply:
Thanks @ShivangiReja Then I will just show the description without any tag. like `/// path: string, # Optional. Location of the document or folder`

Reply:
resolved